alertscenarios/under_replicated_partitions.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python3
```

Comment:
```suggestion
#!/usr/bin/env python3
```

this should run with python3 and pick up the python3 binary wherever its located